src/components/RelatedPosts/styles.module.css

```
@@ -0,0 +1,147 @@
+.wrapper {
+  margin-top: 3rem;
+  padding-top: 2rem;
+  border-top: 1px solid #e5e5e5;
+}
+
+[data-theme='dark'] .wrapper {
+  border-top: 1px solid #262626;
+}
+
+.heading {
+  font-size: 1rem;
```

Stylelint (17.7.0)

[error] Duplicate property "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)